en/snap/advanced-feature.md

```diff
@@ -793,7 +792,7 @@ curl -X POST \
 The sample Snap payment page with online installment feature is displayed below.
 ![snap online installment](./../../asset/image/snap-adv-online-installment.png)
 
-?> ***Note:*** Installment term will show in the Snap payment page after the customer enters the installment-compatible credit card number. For testing this method, refer to [Sandbox test card](/en/technical-reference/sandbox-test.md#card-number).
+?> ***Note:*** Installment term will show in the Snap payment page after the customer enters the installment-compatible credit card number. For testing this method, refer to [Sandbox test card](/en/technical-reference/sandbox-test.md#card number).
```

Comment:
Why is this link broken now? you fix 1 but add 1 other issue. Please fix carefully. @madhura-kulkarni-mh